package.json

```diff
@@ -46,5 +45,13 @@
 			"<rootDir>/test/unit/setup-blocks.js",
 			"<rootDir>/test/unit/setup-wp-aliases.js"
 		]
-	}
+	},
+	"dependencies": {
+		"autoprefixer": "^8.3.0"
```

Comment:
`dependencies` looks like a weird place for it instead of `devDependencies` is it mentioned on their doc?

Reply:
Okay, my bad. It's also working fine with `devDependencies`.
Updated in latest commit.